docs/source/posts/2022/2022-05-23-first-post-mohamed.rst

```diff
@@ -0,0 +1,92 @@
+My journey till getting accepted into GSoC22
+================================================================================
+
+
+.. post:: May 23 2022
+   :author: Mohamed Abouagour 
+   :tags: google
+   :category: gsoc
+
+A Little About Myself
+~~~~~~~~~~~~~~~~~~~~~
+
+My name is Mohamed and I'm from Egypt. I am pursuing a Bachelor of Engineering in Computer Engineering and Automatic Control (expected: 2023), Tanta University, Egypt. 
+
+I've been around computers since 2008 when I had my first PC with 128MB RAM and ran on Windows XP (that's almost all I could remember about it). 
+
+Around 2013, I had some questions about how games are made and how to make them myself!.
+There was no one to answer these questions for me.
+
+My English wasn't any good and the game development Arabic community was very hard to find on the internet. But eventually, I came across a forum with some people speaking about some stuff such as game engines, 3D models, textures, animations, and a lot of fun stuff. That made sense back then and answered some of my questions. Then It was time to get involved with this amazing community. I was lucky enough to start with 3ds Max 2009 Edition, with that little view cube begging me to touch it. It was just love at first sight).
```

Comment:
`sight)`  Why is there a closing bracket here ?

Reply:
> `sight)` Why is there a closing bracket here ?

After modifying the sentence, I accidentally left it out.